app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError

```

Comment:
you should pass descriptive messages when you raise the exception in the visit_cafe method

Reply:
> you should pass descriptive messages when you raise the exception in the visit_cafe method

But in the task w don't have information about description of the error. Or I make a mistake?

Reply:
we have)
https://prnt.sc/co7RFtR2fUeL

Reply:
aaa, ok.
Because, I looked on examples only, thanks